src/main/java/org/jabref/gui/externalfiles/DownloadFullTextAction.java

```diff
@@ -144,8 +144,7 @@ private void addLinkedFileFromURL(BibDatabaseContext databaseContext, URL url, B
                     databaseContext,
                     Globals.TASK_EXECUTOR,
                     dialogService,
-                    preferences.getXmpPreferences(),
-                    preferences.getFilePreferences(),
+                    preferences,
```

Comment:
Why was this change necessary? We should focus the preferences passed around - and not enlarge the scope.